eval/requirements.txt

```
@@ -0,0 +1,14 @@
+FastChat
+GitPython>=3.1.42,<4.0.0
+shortuuid
+openai>=1.13.3,<2.0.0
+psutil
+lm-eval>=0.4.3
+boto3
+requests
+torch
+transformers
+accelerate
+pandas
+pandas-stubs
+kfp[kubernetes]
```

The pipeline component (the container image) needs to contain `kfp[kubernetes]`?

The rest seems to be the same as https://github.com/instructlab/eval/blob/main/requirements.txt, except `boto3` and `requests`.

The dependencies that area already stated in https://github.com/instructlab/eval/blob/main/requirements.txt has been previously already installed when you ran

```
pip install --no-cache-dir git+https://github.com/instructlab/eval
```

Maybe we can reduce this to:

```suggestion
boto3
requests
```

---

I dunno why but these were required to make the workflow proceed - I'm reworking everything so will look at that with new versions